sdk/identity/azure-identity/azure/identity/_credentials/azd_cli.py

```diff
@@ -117,7 +120,11 @@ def get_token(self, *scopes: str, **kwargs: Any) -> AccessToken:
         commandString = " --scope ".join(scopes)
         command = COMMAND_LINE.format(commandString)
         tenant = resolve_tenant(
```

Comment:
I don't see resolve_tenant honors `claims`?

Reply:
That's right; it doesn't use `claims`, but it would have gotten the argument via `**kwargs` in the past. I passed along `claims` and `tenant_id` wherever `**kwargs` are passed to preserve past behavior. I understand if you'd prefer removing `claims` from `resolve_tenant` though, since we know it shouldn't have been used anyway.

Reply:
Let's add it only when we use it to make it more readable.

We don't want to think hard and try to figure out why we have such code after a year passes. :)